### DISCUSSION TITLE
付款后，还提示到期需要续费

### DISCUSSION THREAD

**Discussion by zhouyb126** [getlantern/lantern-forum-cn#565] [General]
付款后，登录专业版，仍然提醒专业版到期需要续费。
当时我专业版的邮箱是： 282738703@qq.com
付款凭证见下图，请帮忙解决一下, 谢谢！
![WechatIMG392](https://github.com/getlantern/lantern-forum-cn/assets/38254265/6fe410ea-36e3-4f71-9122-7a49ff62cee3)

**Comment by MaxMKMax** [comment]
@zhouyb126 您好. 嗯嗯. 好的. 我帮您提交问题. 但我们负责人要下周二才回来了, 请您稍微等一下. 很抱歉啊.

**Comment by zhouyb126** [comment]
恩，好的，麻烦尽快，多谢发自我的 iPhone在 2024年6月13日，18:08，Mike K ***@***.***> 写道：﻿
@zhouyb126 您好. 嗯嗯. 好的. 我帮您提交问题. 但我们负责人要下周二才回来了, 请您稍微等一下. 很抱歉啊.

**Reply by MaxMKMax** [reply]
@zhouyb126 好的. 谢谢您的体谅. 我这里会尽快帮您催. 一有消息马上通知您!

**Comment by zhouyb126** [comment]
你好，我是续费的，买的是2年，应该给的是2年+3个月，给你日期不对，麻烦帮忙确认下发自我的 iPhone在 2024年6月13日，19:18，Mike K ***@***.***> 写道：﻿
@zhouyb126 好的. 谢谢您的体谅. 我这里会尽快帮您催. 一有消息马上通知您!

**Reply by MaxMKMax** [reply]
@zhouyb126 好的. 我帮您提交问题.

**Comment by MaxMKMax** [comment]
@zhouyb126 
您好. 这是您的激活码. 不好意思让您久等了.
FGYMW-WBBXT-K7VDF-JD7QV-P8BG7

**Comment by MaxMKMax** [comment]
@zhouyb126 您好. 请问您可以登录成功了吗? 您的问题解决了吗?

**Comment by zhouyb126** [comment]
您好，麻烦帮我把剩下的退费吧，谢谢发自我的 iPhone在 2024年6月13日，18:37，Qq ***@***.***> 写道：﻿恩，好的，麻烦尽快，多谢发自我的 iPhone在 2024年6月13日，18:08，Mike K ***@***.***> 写道：﻿
@zhouyb126 您好. 嗯嗯. 好的. 我帮您提交问题. 但我们负责人要下周二才回来了, 请您稍微等一下. 很抱歉啊.

**Comment by zhouyb126** [comment]
发自我的 iPhone在 2025年1月17日，11:17，Qq ***@***.***> 写道：﻿您好，麻烦帮我把剩下的退费吧，谢谢发自我的 iPhone在 2024年6月13日，18:37，Qq ***@***.***> 写道：﻿恩，好的，麻烦尽快，多谢发自我的 iPhone在 2024年6月13日，18:08，Mike K ***@***.***> 写道：﻿
@zhouyb126 您好. 嗯嗯. 好的. 我帮您提交问题. 但我们负责人要下周二才回来了, 请您稍微等一下. 很抱歉啊.

**Comment by zhouyb126** [comment]
这是当时的付款单，麻烦帮我剩下的退款吧，多谢发自我的 iPhone在 2025年1月17日，11:17，Qq ***@***.***> 写道：﻿您好，麻烦帮我把剩下的退费吧，谢谢发自我的 iPhone在 2024年6月13日，18:37，Qq ***@***.***> 写道：﻿恩，好的，麻烦尽快，多谢发自我的 iPhone在 2024年6月13日，18:08，Mike K ***@***.***> 写道：﻿
@zhouyb126 您好. 嗯嗯. 好的. 我帮您提交问题. 但我们负责人要下周二才回来了, 请您稍微等一下. 很抱歉啊.

**Comment by zhouyb126** [comment]
你好，麻烦帮我处理下退款发自我的 iPhone在 2025年1月17日，11:22，Qq ***@***.***> 写道：﻿<image0.png>这是当时的付款单，麻烦帮我剩下的退款吧，多谢发自我的 iPhone在 2025年1月17日，11:17，Qq ***@***.***> 写道：﻿您好，麻烦帮我把剩下的退费吧，谢谢发自我的 iPhone在 2024年6月13日，18:37，Qq ***@***.***> 写道：﻿恩，好的，麻烦尽快，多谢发自我的 iPhone在 2024年6月13日，18:08，Mike K ***@***.***> 写道：﻿
@zhouyb126 您好. 嗯嗯. 好的. 我帮您提交问题. 但我们负责人要下周二才回来了, 请您稍微等一下. 很抱歉啊.

**Comment by zhouyb126** [comment]
您好， 麻烦帮忙处理下退款问题，我邮件好多次了（282738703@qq.com）， 多谢，

**Comment by MaxMKMax** [comment]
@zhouyb126 您好. 使用超过一个月我们就无法给您进行退款了. 不好意思啊. 谢谢您的理解!
